frontend/components/course-listings/evaluationList.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
The equivalent of this file that you are doing is this https://github.com/uwasystemhealth/IndEAA/blob/develop/client/components/Coordinator/Evaluation/EvaluationList.js

Reply:
I have changed the file names so they are more intuitive and are capitalized, was there anything else about the file structure that you wanted me to change?

Reply:
I was refering to the name of the file `evaluationList` -> `EvaluationList`... and you should stay consistent with the folder names to `course-listings` -> `courseListings`.

Reply:
ah looks like you've addressed it... looks good!